procurement_purchase_no_grouping/models/res_config_settings.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 - Radovan Skolnik <radovan@skolnik.info>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    """Configuration of default value for procurement purchase grouping."""
+
+    _inherit = "res.config.settings"
+    # _name = "procurement.purchase.grouping.settings"
+    _description = "Procurement purchase grouping settings"
+
+    default_procured_purchase_grouping = fields.Selection(
+        [
+            ("standard", "Standard grouping"),
+            ("line", "No line grouping"),
+            ("order", "No order grouping"),
+        ],
+        string="Procured purchase grouping",
+        default="standard",
+        default_model="product.category",
```

Comment:
You should store this other way, as this means that new categories will take this value instead of empty one, which can't be the desired behavior, as it's not the same having "system wide grouping" than the last default that it was. I think a better granularity is to have this selection at res.company, at put here a related field.

Reply:
> You should store this other way, as this means that new categories will take this value instead of empty one, which can't be the desired behavior, as it's not the same having "system wide grouping" than the last default that it was. I think a better granularity is to have this selection at res.company, at put here a related field.

@pedrobaeza will (try to) do. But does this really solve the failed tests? In them it seems as if the system settings in some of the tests wasn't there at all.

Reply:
Well, other approach, other tests, so it's worth trying. Anyway, when you have it, I'll see specifically possible problems